### PR TITLE
linking default value

### PR DESCRIPTION
**Pull request by olsonpm**

### CODE REVIEW

**Comment by dougwilson** [comment]
>  Z_DEFAULT_COMPRESSION requests a default compromise between speed and compression (currently equivalent to level 6). 

We can just add that directly into the readme, rather than a esoteric link :)

**Comment by Fishrock123** [comment]
Hmmm, I thought it defaulted to level 7?

**Comment by dougwilson** [comment]
> Hmmm, I thought it defaulted to level 7?

Maybe. I only took it from the linked documentation, which may or may not apply to the bundled version of zlib in Node.js. It's easy to test: compress something with -1, then compress it with 6. Are they the same bytes? Then that's the default :)

**Comment by Fishrock123** [comment]
Hmm, looks like it should be 6. My bad.

**Comment by olsonpm** [comment]
Yeah, i didn't include the number because i thought that was the only reason to 
disclude it in the first place.

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ compression level".
   - `8`
   - `9` Best compression (also `zlib.Z_BEST_COMPRESSION`).
 
-The default value is `zlib.Z_DEFAULT_COMPRESSION`, or `-1`.
+The default value is `zlib.Z_DEFAULT_COMPRESSION`, or `-1`.  The actual level specified by default cab be found [here](http://www.zlib.net/manual.html).
```
Comment (jonathanong) [inline README.md]: can*

**Pull request closed without merging**